src/main/java/com/unosquare/cvgenerator/controller/HelloController.java

```diff
@@ -17,8 +17,8 @@
 
     @GetMapping("/hello")
     public String sayHello(Model theModel) {
-        GreetingModelView greetingModelView = greetingService.findById(1);
-        theModel.addAttribute("greeting", greetingModelView);
+        GreetingDTO greetingDTO = greetingService.findById(1);
```

Comment:
Future change: can be updated when adding the proper endpoints, but just making a reminder to remove hardcoded id:
```suggestion
        GreetingDTO greetingDTO = greetingService.findById(id);
```